### PR TITLE
Removed runtime results

### PR DESCRIPTION
**Pull request by 1bk**
To show actual results instead.

### CODE REVIEW

File: call_times.py [hunk 1]
```diff
@@ -27,7 +27,7 @@ def overlap(time, callTimes):
             max_agents = max_n
         time += 1
     
-    print('Runtime:', (tt.time() - start_time)*1000)
+    //print('Runtime:', (tt.time() - start_time)*1000)
```
Comment (mavx) [inline call_times.py]: Comments in Python starts with `#` instead of `//`

**Review by mavx — CHANGES_REQUESTED** [review]
Please ensure correct syntax before submitting pull request :D

File: call_times.py (hunk 1 above)
Comment (1bk) [reply call_times.py]: Sorry, just played around in C recently 😅

**Review by mavx — APPROVED** [review]
Good job!

**Pull request merged**